src/index.js

```diff
@@ -54,25 +54,81 @@ const metrics = {
+		help: "Speed Index, more info here: https://web.dev/speed-index",
+		labelNames: [ 'page' ]
+	}),
+	mobile_max_potential_fid: new Prometheus.Gauge({
+		name: 'pagespeed_mobile_max_potential_fid_seconds',
+		help: "The maximum potential First Input Delay that your users could experience is the duration, in milliseconds, of the longest task. https://developers.google.com/web/updates/2018/05/first-input-delay",
+		labelNames: [ 'page' ]
+	}),
+	mobile_first_meaningful_paint: new Prometheus.Gauge({
+		name: 'pagespeed_mobile_first_meaningful_paint_seconds',
+		help: "First Meaningful Paint measures when the primary content of a page is visible. https://web.dev/first-meaningful-paint",
+		labelNames: [ 'page' ]
+	}),
+	mobile_performance_score: new Prometheus.Gauge({
+		name: 'pagespeed_mobile_performance_score',
+		help: "Performance Score",
+		labelNames: [ 'page' ]
+	}),
+	mobile_accessibility_score: new Prometheus.Gauge({
+		name: 'pagespeed_mobile_accessibility_score',
+		help: "Accessibility Score",
+		labelNames: [ 'page' ]
 	})
 };
 
 for (let page of pages) {
 	setInterval(async () => {
-		let data = await pagespeed(page, apiKey);
-		if (data != null) {
+		let desktopData = await pagespeed(page, apiKey, 'DESKTOP');
+		if (desktopData != null) {
 			try {
-				metrics.first_contentful_paint.set({ page }, dataextractor.first_contentful_paint(data));
-				metrics.first_cpu_idle.set({ page }, dataextractor.first_cpu_idle(data));
-				metrics.interactive.set({ page }, dataextractor.interactive(data));
-				metrics.speed_index.set({ page }, dataextractor.speed_index(data));
-				metrics.max_potential_fid.set({ page }, dataextractor.max_potential_fid(data));
-				metrics.first_meaningful_paint.set({ page }, dataextractor.first_meaningful_paint(data));
-				metrics.performance_score.set({ page }, dataextractor.performance_score(data));
-				metrics.accessibility_score.set({ page }, dataextractor.accessibility_score(data));
+				metrics.first_contentful_paint.set({ page }, dataextractor.first_contentful_paint(desktopData));
+				metrics.first_cpu_idle.set({ page }, dataextractor.first_cpu_idle(desktopData));
+				metrics.interactive.set({ page }, dataextractor.interactive(desktopData));
+				metrics.speed_index.set({ page }, dataextractor.speed_index(desktopData));
+				metrics.max_potential_fid.set({ page }, dataextractor.max_potential_fid(desktopData));
+				metrics.first_meaningful_paint.set({ page }, dataextractor.first_meaningful_paint(desktopData));
+				metrics.performance_score.set({ page }, dataextractor.performance_score(desktopData));
+				metrics.accessibility_score.set({ page }, dataextractor.accessibility_score(desktopData));
 			} catch (e) {
 				console.error(`data parsing failed, response dumped, url called: ${page}`);
-				fs.writeFileSync('response.json', data, { mode: 0o755 });
+				fs.writeFileSync('response.json', desktopData, { mode: 0o755 });
+			}
+		}
+		let mobileData = await pagespeed(page, apiKey, 'MOBILE');
```

Comment:
Rather than having the same code twice we could do something like `['DESKTOP', 'MOBILE'].each((i)=>{ doStuffWith i; })`. Thoughts?

Reply:
Did wonder about that and wasn't sure it was worth it for just 2 use cases but will have a look into it more 😄

Reply:
Had a quick think about this one, and as all the gauges need to be uniquely named (and the desktop ones need to stay as is for current services that are using them) couldn't immediately think of a straightforward way of doing this. I'm sure it's possible but maybe quite a big refactor. Will sleep on it and have another think tomorrow.